recipe/meta.yaml

```diff
@@ -1,31 +1,21 @@
-{% set version = "3.8.0" %}
-{% set build = 1 %}
-
-# ensure mpi is defined (needed for conda-smithy recipe-lint)
-{% set mpi = mpi or 'nompi' %}
-
-{% if mpi == 'nompi' %}
-# prioritize nompi variant via build number
-{% set build = build + 100 %}
-{% endif %}
+{% set version = "3.9.0" %}
+{% set name = "arpack" %}
+{% set build = 0 %}
 
 package:
-  name: arpack
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: arpack-{{ version }}.tar.gz
-  url: https://github.com/opencollab/arpack-ng/archive/{{ version }}.tar.gz
-  sha256: ada5aeb3878874383307239c9235b716a8a170c6d096a6625bfd529844df003d
+  url: https://github.com/opencollab/{{ name }}-ng/archive/{{ version }}.tar.gz
+  sha256: 24f2a2b259992d3c797d80f626878aa8e2ed5009d549dad57854bbcfb95e1ed0
 
 build:
   number: {{ build }}
+  skip: true  # [blas_impl == "mkl"]
```

Comment:
Could you either remove the references to `blas_impl` (and therefore `mkl`) entirely, or add a comment saying we are temporarily skipping MKL builds?

Reply:
Currently does not build for mkl. If I remove skip, it will try to build for it and will fail. Therefore, gonna add a comment.